tests/e2e/pageObjects/browser-page.ts

```diff
@@ -22,6 +22,8 @@ export class BrowserPage {
     //*The following categories are ordered alphabetically (Alerts, Buttons, Checkboxes, etc.).
     //-------------------------------------------------------------------------------------------
     //BUTTONS
+    streamDeleteButton = Selector(`[data-testid="stream-delete-btn"]`);
```

Comment:
do you need quotes here?
looks like it can be just `[data-testid=stream-delete-btn]`